### DISCUSSION TITLE
how do I use the ExtHang3r

### DISCUSSION THREAD

**Discussion by reowl044** [3kh0/ext-remover#1191] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
how do I use the ExtHang3r

**Comment by 88411** [comment]
Yeah, whenever I try to use it it crashes my whole Chromebook

**Reply by reowl044** [reply]
but still how do i use it

**Reply by 88411** [reply]
I mean just follow the instructions. Do you not know how to open it or do you just not understand stand the instructions?

**Reply by 88411** [reply]
Just copy the code into the url bar

**Reply by reowl044** [reply]
all it says is this
![Screenshot 2024-10-16 1 25 58 PM](https://github.com/user-attachments/assets/58a0219c-7d64-4bcc-9ec6-330b22649dc9)

**Reply by VLHertzberg** [reply]
What kind of extensions do you have?

**Reply by schoolexploitkid** [reply, marked as answer]
You don't have the correct extensions to use it. Try another exploit.
